source/js/main.js

```diff
@@ -221,25 +220,8 @@ let main = {
       }));
     }
 
-    // Show Takeover for new visitors
-    if (env.SHOW_TAKEOVER && !Cookies.get(`seen-takeover`) && document.querySelector(`#view-home .takeover`)) {
-      let elWrapper = document.querySelector(`#view-home > .wrapper`);
-
-      // Don't allow the content block to scroll
-      elWrapper.style.overflow = `hidden`;
-      elWrapper.style.height = `100vh`;
-
-      let onTakeoverHide = () => {
-        // Allow scrolling again when the takeover is dismissed
-        elWrapper.style.overflow = null;
-        elWrapper.style.height = null;
-      };
-
-      apps.push(new Promise(resolve => {
-        ReactDOM.render(<Takeover onHide={onTakeoverHide} whenLoaded={() => resolve()}/>, document.querySelector(`#view-home .takeover`));
-        Cookies.set(`seen-takeover`, `true`, { expires: 365 });
-        ReactGA.pageview(`/welcome-splash`);
-      }));
+    if (Cookies.get(`seen-takeover`)) {
```

Comment:
do we need to do this?

Reply:
I would say no: @alanmoo, @Pomax can one of you confirm?

Reply:
I...don't think so?